src/Text/Text.stories.js

```diff
@@ -166,6 +66,37 @@ storiesOf("Text", module)
     };
   })
   .addWithChapters("Attention text", () => {
+    const customText = text(
+      "Text",
+      "Sixth can't place isn't fly you're make Shall day, yielding earth made greater stars own you'll earth can't." +
+        " Night grass great. Fly firmament over. Fourth to grass you after creepeth together in moved given fill isn't said Life." +
+        " Male signs unto form kind Brought signs heaven, in have wherein. Isn't. Behold meat living you're evening very replenish you'll. Under hath.",
+    );
+    return {
+      info:
+        "The most basic component for rendering text blocks. Visit Orbit.Kiwi for more detailed guidelines.",
+      chapters: [
+        {
+          sections: [
+            {
+              sectionFn: () => <Text type="attention">{customText}</Text>,
+              options,
+            },
+          ],
+        },
+      ],
+    };
+  })
+  .addWithChapters("Playground", () => {
+    const type = select(
+      "Type",
+      {
+        primary: "primary",
+        secondary: "secondary",
+        attention: "attention",
+      },
+      "attention",
+    );
     const element = select(
```

Comment:
```
<p> <div> <span>
```

may have different shared behavior from browser, like `display: block` please always override it, or is this on purpose ?

Reply:
Talked with @darkwindcz and we can't decide now. We want to leave it on browser for now.